.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vscode
+dist
```

Comment:
and remove the dist from `gitignore` too as no bundle is being created by any of the packages

Reply:
Yup :)